lti_emailer/settings/local.py

```diff
@@ -15,3 +15,14 @@
 }
 
 dictConfig(LOGGING)
+
+CANVAS_URL = 'http://canvas.icommons.harvard.edu/'
```

Comment:
This isn't necessary -- it comes from `base.py`; to change it, you update your secure settings (see https://github.com/Harvard-University-iCommons/lti_emailer/blob/develop/lti_emailer/settings/base.py#L266). Let's avoid duplication/multiple places to update things.